modules/gltf/src/gltf/gltf-parser.js

```diff
@@ -206,10 +224,29 @@ export default class GLTFParser {
 
   // PARSING HELPERS
 
+  _getFullUri(uri, base) {
+    if (uri.startsWith('data:') || uri.startsWith('http:') || uri.startsWith('https:')) {
+      return uri;
+    }
+
+    return base.substr(0, base.lastIndexOf('/') + 1) + uri;
+  }
+
   // Start loading linked assets
   _loadLinkedAssets(options) {
-    // TODO: Not implemented
-    // TODO: Return a promise?
+    const promises = [];
+
+    if (!this.glbParser) {
+      this.gltf.buffers.forEach(buffer => {
+        if (buffer.uri) {
```

Comment:
Would prefer if we handle base64 encoded files in the non-async path?

Either fix or add TODO.